modules/ovirt-csi-driver-storage-class.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+
+[id="ovirt-csi-driver-storage-class_{context}"]
+= oVirt CSI driver storage class
+
+OpenShift Container Platform will create a default `StorageClass` *ovirt-csi-sc* which is used for creating dynamically provisioned persistent volumes (for information on storage classes see link:https://deploy-preview-28872--osdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi.html#csi-dynamic-provisioning_persistent-storage-csi[Dynamic Provisioning]).
+To create additional storage classes for different configurations, create and save a file with the `StorageClass` object described by the following YAML:.
+
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: <STORAGE-CLASS-NAME> [1]
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "false" [2]
+provisioner: csi.ovirt.org
+parameters:
+    storageDomainName: <RHV-STORAGE-DOMAIN-NAME> [3]
+    thinProvisioning: "true" [4]
+    csi.storage.k8s.io/fstype: ext4  [5]
+----
++
+<1> Name of the storage class
+<2> If the storage class is the default storage class in the cluster. This should be false in most cases. If you choose to set it to true, the existing default storage class should be edited and set to false.
```

Comment:
I don't understand, user can change the default storage state to true? or only can change different storage class?

Reply:
This is a technical question that I don't have an answer for...